### PR TITLE
feature: cifar10 instead of mnist

### PR DESCRIPTION
**Pull request by FelixAbrahamsson**
Will make some PRs onto this one with
- Feature -> StandardizedImage change
- resize in Feature and datastream
- torch 1.7.1 + persistent workers

### CODE REVIEW

File: {{cookiecutter.repository_name}}/{{cookiecutter.package_name}}/architecture/feature.py [hunk 1]
```diff
@@ -8,9 +8,7 @@
 
 
 def standardized(image: Image.Image):
-    return (
-        torch.stack([torch.as_tensor(np.array(image, dtype=np.float32))]) / 255 * 2 - 1
-    )
+    return torch.as_tensor(np.array(image, dtype=np.float32)).permute(2, 0, 1) / 255
```
Comment (samedii) [inline {{cookiecutter.repository_name}}/{{cookiecutter.package_name}}/architecture/feature.py:11]: I think we should standardize between -1 and 1

File: {{cookiecutter.repository_name}}/{{cookiecutter.package_name}}/architecture/model.py [hunk 2]
```diff
@@ -11,14 +11,14 @@ class Model(nn.Module):
     def __init__(self):
         super().__init__()
         self.logits = ModuleCompose(
-            nn.Conv2d(1, 32, 3, 1),
```
Comment (samedii) [inline {{cookiecutter.repository_name}}/{{cookiecutter.package_name}}/architecture/model.py:14]: Could be nice to use a torchvision classifier since we are looking at cifar10 now and not mnist?

**Review by samedii — APPROVED** [review]
Nice!

File: {{cookiecutter.repository_name}}/{{cookiecutter.package_name}}/architecture/feature.py (hunk 1 above)
Comment (samedii) [reply {{cookiecutter.repository_name}}/{{cookiecutter.package_name}}/architecture/feature.py:11]: (Unless we are using something pretrained)

File: {{cookiecutter.repository_name}}/{{cookiecutter.package_name}}/architecture/model.py (hunk 2 above)
Comment (FelixAbrahamsson) [reply {{cookiecutter.repository_name}}/{{cookiecutter.package_name}}/architecture/model.py:14]: Yeah sure :) Makes more sense with the 0-1 normalization then

Comment (samedii) [reply {{cookiecutter.repository_name}}/{{cookiecutter.package_name}}/architecture/model.py:14]: I think I dislike having pretrained weights for this since we will need to normalize with their weights etc. Feels too specific for a template

Comment (FelixAbrahamsson) [reply {{cookiecutter.repository_name}}/{{cookiecutter.package_name}}/architecture/model.py:14]: Okay, I thought it made sense because it feels like we use something pretrained more often than not. But I can see the argument for not having it.

Comment (FelixAbrahamsson) [reply {{cookiecutter.repository_name}}/{{cookiecutter.package_name}}/architecture/model.py:14]: Actually after thinking about it I'd prefer having an even smaller model than the current one. Takes less time to run tests and the model will be overwritten anyway.
